packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -2660,25 +2645,6 @@ function unwindUnitOfWork(unitOfWork: Fiber): void {
       returnFiber.deletions = null;
     }
 
-    if (revertRemovalOfSiblingPrerendering) {
-      // If there are siblings, work on them now even though they're going to be
-      // replaced by a fallback. We're "prerendering" them. Historically our
-      // rationale for this behavior has been to initiate any lazy data requests
-      // in the siblings, and also to warm up the CPU cache.
-      // TODO: Don't prerender siblings. With `use`, we suspend the work loop
-      // until the data has resolved, anyway.
-      const siblingFiber = incompleteWork.sibling;
-      if (siblingFiber !== null) {
-        // This branch will return us to the normal work loop.
-        workInProgress = siblingFiber;
-        return;
-      }
-    } else {
-      // NOTE: If we re-enable sibling prerendering in some cases, this branch
```

Comment:
Same here, I'd like to keep this comment around for context

Reply:
makes sense. I added the comment back.